src/wp-includes/blocks.php

```diff
@@ -922,6 +922,10 @@ function insert_hooked_blocks( &$parsed_anchor_block, $relative_position, $hooke
 		 */
 		$parsed_hooked_block = apply_filters( "hooked_block_{$hooked_block_type}", $parsed_hooked_block, $hooked_block_type, $relative_position, $parsed_anchor_block, $context );
 
+		if ( null === $parsed_hooked_block ) {
```

Comment:
Ok, the code snippet was folded 😄

It means that it brings consistency into how these filters work with the ability to return `null`. All good 👍🏻

Reply:
Not sure I'm following; can you rephrase? 😅 

The filter can only modify the `$parsed_hooked_block`; the `$hooked_block_type` is there in case the `$parsed_hooked_block` is e.g. wrapped in a Group block, in which case the top-level `$parsed_hooked_block['blockName']` will change (whereas the `$hooked_block_type` is kept unchanged so that it can be used to set `ignoredHookedBlocks` correctly).